### ISSUE TITLE
The Markdown -> HTML conversion script gets some stuff in parentheses wrong

### ISSUE THREAD

**Issue by foolip** [whatwg/sg#45]
https://github.com/whatwg/whatwg.org/pull/127 fixed a case like this, I found another.

@othermaciej, I don't know where the script is, but assigning to you to sort out or open another issue on the right repo.

**Comment by foolip** [comment]
I've run all the new files through https://validator.w3.org/checklink so https://github.com/whatwg/whatwg.org/pull/128 is the last of it, just need to make sure the script does the right thing before regenerating.

**Comment by othermaciej** [comment]
The script is not in a public repository yet. It will likely end up either here or in the whatwg.org repo, depending on how we fit it into the whatwg.org website build process.

**Comment by othermaciej** [comment]
I've proposed a PR to add the script to whatwg.org: https://github.com/whatwg/whatwg.org/pull/132

Once that lands, I think the issue should move there. My understanding of the issue is that parens after brackets should not be treated as a link if there is an intervening space. Global replace of "] (" to "] \(" as a preprocessing step should do it, I think.

**Comment by foolip** [comment]
Moved to https://github.com/whatwg/whatwg.org/issues/134.
